src/Accelerators/NNPA/Conversion/ZLowToLLVM/ZLowToLLVMCommon.cpp

```diff
@@ -215,6 +217,7 @@ ZTensor ZTensorHelper::getZTensor(Value bufferPtr, zdnn_data_types dataType,
   Value transformedDescPtr =
       getTransformedDescPtr(preTransformedDescPtr, isConcat, concatInfo);
   // Create the input zTensor.
+  // TODO: evaluate if a heap alloc would not be better.
```

Comment:
I often see in MLIR they use alloca to create a LLVM struct.

Reply:
Make sense, it's still small. Want me to remove the comment? @tungld

Reply:
Yes, it's ok to remove it. Thanks!

Reply:
Done.